### DISCUSSION TITLE
source-map behaves abnormally, please how to solve it

### DISCUSSION THREAD

**Discussion by 0Ams** [nuxt/nuxt#19973] [Questions (Nuxt 2)]
I am trying to debug using Visual Studio Code. I have the debugger launching properly using Chrome debug tools, and properly using a map, but when I try to place breakpoints in my .js or .vue files, VS Code seems to be placing the breakpoints in the wrong place. For example, set breakpoint method name `getInfo` in Original source file but completely different from where I set in build file

# Information 
```
  Binaries:
    Node: 18.15.0
    npm: 9.5.0
  Browsers:
    Chrome
  npmPackages:
    nuxt: ~2.16.3 => 2.16.3 
```
# launch.json
```
{
  "type": "chrome",
  "request": "launch",
  "name": "Launch Chrome",
  "url": "http://localhost:8080",
  "webRoot": "${workspaceRoot}",
  "sourceMaps": true,
  "sourceMapPathOverrides": {
            "webpack:///./*": "${webRoot}/*"
  }
}
```

When I thought about it, the behavior of source-map looks strange. I'm not getting the location of the built files normally through the location of the existing code, so I'm asking if you can help me.

**Comment by baiwusanyu-c** [comment]
I have the same problem, which makes me unable to debug nuxt source code, did you finally solve it? Or is there a better way to debug

**Comment by baiwusanyu-c** [comment]
I get it !
1.pnpm run build:stub 
2. pnpm run dev
